pandas/core/generic.py

```diff
@@ -5458,13 +5458,63 @@ def asof(self, where, subset=None):
     # Action Methods
 
     _shared_docs['isna'] = """
+        Detect missing values.
+
         Return a boolean same-sized object indicating if the values are NA.
+        NA values, such as None or :attr:`numpy.NaN`, get mapped to True
+        values.
+        Everything else get mapped to False values. Characters such as empty
+        strings `''` or :attr:`numpy.inf` are not considered NA values
+        (unless you set :attr:`pandas.options.mode.use_inf_as_na` `= True`).
```

Comment:
Does this `attr` work? I don't know if its in our API docs. I think it'd be ok to jsut have unless you set \`\`pandas.options.mode.use_inf_as_na = True\`\`

Reply:
We have seen :attr: references throughout other docstrings (frame.py). I can change it, thoughts?

Reply:
It's fine.

Reply:
It's not the `attr` itself, but the fact there is nothing to link to I think?

I would rather make this ``` ``pandas.options.mode.use_inf_as_na = True`` ```